packages/tasit-sdk/src/testHelpers/EstateRegistry.json

```diff
@@ -11939,21 +11836,14 @@
   "legacyAST": {
     "absolutePath": "/home/cmarcelo/git/TasitSDK/packages/tasit-contracts/decentraland/land/contracts/estate/EstateRegistry.sol",
     "exportedSymbols": {
-      "EstateRegistry": [
-        848
-      ]
+      "EstateRegistry": [848]
     },
     "id": 849,
     "nodeType": "SourceUnit",
     "nodes": [
       {
         "id": 16,
-        "literals": [
-          "solidity",
-          "^",
-          "0.4",
-          ".23"
-        ],
+        "literals": ["solidity", "^", "0.4", ".23"],
```

Comment:
Hmmm, It occurred because of overloading function removal.
We can't ignore `*.json` and we can't ignore `testHelpers/*` either. (I've used correctly this time?)
I'm not sure about ignoring `testHelpers/*json`. Should we move all ABIs files to `abi/` dir and ignore `**/abi/*` ?

Reply:
Done

Reply:
Great. Resolving!